api/info/service.go

```diff
@@ -312,11 +312,20 @@ type UptimeRequest struct {
 	SubnetID ids.ID `json:"subnetID"`
 }
 
+// Deprecated: Uptime is deprecated for Subnet Validators.
+// It will be available only for Primary Network Validators.
```

Comment:
I think this applies to a number of locations throughout this PR:

This marks the whole method as deprecated. I think we should only be deprecating the `SubnetID` field inside of the `UptimeRequest` struct.

Reply:
I don't know if there is a way to deprecate only a single param for functions like `NodeUptime(subnetID ids.ID)`. I think it's fine to omit functions and deprecate APIs. What do you say?